src/coreclr/System.Private.CoreLib/src/System/Reflection/TypeNameResolver.CoreCLR.cs

```diff
@@ -128,14 +132,20 @@ internal static RuntimeType GetTypeReferencedByCustomAttribute(string typeName,
 
         // Used by VM
         internal static unsafe RuntimeType? GetTypeHelper(char* pTypeName, RuntimeAssembly? requestingAssembly,
-            bool throwOnError, bool requireAssemblyQualifiedName)
+            bool throwOnError, bool requireAssemblyQualifiedName, IntPtr unsafeAccessorMethod)
         {
             ReadOnlySpan<char> typeName = MemoryMarshal.CreateReadOnlySpanFromNullTerminated(pTypeName);
-            return GetTypeHelper(typeName, requestingAssembly, throwOnError, requireAssemblyQualifiedName);
+            return GetTypeHelper(typeName, requestingAssembly, throwOnError, requireAssemblyQualifiedName, unsafeAccessorMethod);
         }
 
         internal static unsafe RuntimeType? GetTypeHelper(ReadOnlySpan<char> typeName, RuntimeAssembly? requestingAssembly,
             bool throwOnError, bool requireAssemblyQualifiedName)
+        {
+            return GetTypeHelper(typeName, requestingAssembly, throwOnError, requireAssemblyQualifiedName, IntPtr.Zero);
+        }
+
+        private static unsafe RuntimeType? GetTypeHelper(ReadOnlySpan<char> typeName, RuntimeAssembly? requestingAssembly,
+            bool throwOnError, bool requireAssemblyQualifiedName, IntPtr unsafeAccessorMethod)
```

Comment:
```suggestion
        internal static unsafe RuntimeType? GetTypeHelper(ReadOnlySpan<char> typeName, RuntimeAssembly? requestingAssembly,
            bool throwOnError, bool requireAssemblyQualifiedName, IntPtr unsafeAccessorMethod = 0)
```
Just make it an arg with default value instead of wrapper?

Reply:
I really dislike default arguments unless there is a good reason to have them. They make code reviews much harder, but if that is preferred I'll do it.

Reply:
I do not have a strong opinion.

I think it is important to use named arguments when calling methods with a lot of arguments like this one. I do not think there is significant readability difference between one method with default arguments vs. multiple overloads. The calling code looks exactly same in both cases. I tend to go for the more compact variant that is the default argument variant in this case.